File: mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
   int height;

    // ask for height, if invalid height repeat prompt
    do
    {

        height = get_int("Height: ");

    }
    while (height < 0 || height > 23);

    // for loops to print number of hashes and spaces for left and right pyramid with 2 gap in between
    for (int x = 0; x < height; x++)
    {
        for (int s = height - x; s > 1; s--)
        {
            printf(" ");
        }
        for (int h = 0; h < x + 1; h++)
        {
            printf("#");
        }
        printf("  ");
        for (int h = 0; h < x + 1; h++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: mario.py
from cs50 import get_int

while True:
    height = get_int("Height: ")
    if (0 <= height <= 23):
        break

j=0

for i in range(height):

    for j in range(height - i-1, j > 1, -1):
        print(" ", end="")

    for k in range(i + 1):
        print("#", end="")

    print("  ", end="")

    for k in range(i + 1):
        print("#", end="")

    print("")




